pkg/apis/core/types.go

```diff
@@ -2189,7 +2189,7 @@ type SecretKeySelector struct {
 
 // EnvFromSource represents the source of a set of ConfigMaps
 type EnvFromSource struct {
-	// An optional identifier to prepend to each key in the ConfigMap.
+	// An optional identifier to prepend to each key in the ConfigMap or Secret.
```

Comment:
How I'd write it:
```suggestion
	// Optional text to prepend to the name of each environment variable.
```

Reply:
Thanks for your review, updated this to match your recommendation and re-committed